packages/babel-plugin-transform-classes/test/fixtures/extend-builtins/loose/expected.js

```diff
@@ -0,0 +1,23 @@
+function _inheritsLoose(subClass, superClass) { subClass.prototype = Object.create(superClass.prototype); subClass.prototype.constructor = subClass; subClass.__proto__ = superClass; }
+
+var _gPO = Object.getPrototypeOf || function _gPO(o) { return o.__proto__; };
+
+var _sPO = Object.setPrototypeOf || function _sPO(o, p) { o.__proto__ = p; };
+
+var _construct = Reflect.construct || function _construct(Parent, args, Class) { var Constructor, a = [null]; a.push.apply(a, args); Constructor = Parent.bind.apply(Parent, a); return _sPO(new Constructor(), Class.prototype); };
+
+var _cache = typeof WeakMap === "function" && new WeakMap();
+
+function _wrapNativeSuper(Class) { if (_cache) { if (_cache.has(Class)) return _cache.get(Class); _cache.set(Class, Wrapper); } function Wrapper() {} Wrapper.prototype = Object.create(Class.prototype, { constructor: { value: Wrapper, enumerable: false, writeable: true, configurable: true } }); return _sPO(Wrapper, _sPO(function Super() { return _construct(Class, arguments, _gPO(this).constructor); }, Class)); }
```

Comment:
Don't need `enumerate: false` (default, and if omitted, core-js/es5-sham can avoid being forced to throw on ES3 engines)

Reply:
true that it's `false` by default but also true that ES3 engines *should* throw if you extend builtins because ES3 engines are not supported. IE <= 10 is discouraged and that's already mostly ES5

Reply:
The `enumerable: false` is also used by the `_inherits` helper.
Also, ES3 engines don't support `setPrototypeOf`/`__proto__`, so classes won't work in any case.

Reply:
The use of `__proto__` was actually added to the helpers specifically so it would work on ES3 engines that were polyfilled.

The idea was that, even if the engine doesn't support it, as long as we read from `__proto__` first, we're going to get a reference to the relevant constructor, which is all that we need for calling super. static property inheritance will still, of course, be broken.

https://github.com/babel/babel/pull/3527